force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -253,12 +335,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer sumPositiveIntegers() {
         List<Integer> numbers = new List<Integer>{-1, 2, -3, 4, -5, 6}; //DO NOT CHANGE
+        int sum = 0;
 
         // Loop through the list of integers
             // if the number is negative skip this iteration
+        for (integer number : numbers) {
+            if (number <= 0) {
+                continue;
+            }
+            sum += number;
```

Comment:
You really don't need to use a continue block here. Can you solve it with it out?